Tests/Resources/Fixtures/config/full.yml

```diff
@@ -0,0 +1,9 @@
+http_client:
+    main_alias:
```

Comment:
> Isn't .editorconfig included in the repo?

missed it. added now.

> 2 spaces is required in YML files

hm, symfony usually does 4. should we make the bundle look like the
preferences of symfony or like the preferences of php-http?